### PR TITLE
Testing daml models

### PR DESCRIPTION
**Pull request by drsk0**
This is a scenario for testing DAML models with DAML scenarios.

### CODE REVIEW

File: testing-daml-models/testing-with-scenarios/5_scenario_tracing.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+<pre>
+create this with following = userToFollow :: following
+</pre>
+
+with
+
+<pre class="file" data-filename="daml/User.daml" data-target="clipboard">
+      trace "adding new follower" (create this with following = userToFollow :: following)
+</pre>
```
Comment (nemanja-da) [inline testing-daml-models/testing-with-scenarios/5_scenario_tracing.md]: If the [preview link](https://www.katacoda.com/drsk/courses/testing-daml-models/testing-with-scenarios) is the same as before then this still creates an issue with the "Copy to Editor" button

image

Comment (ghost) [reply testing-daml-models/testing-with-scenarios/5_scenario_tracing.md]: so how in the world can I simply show code?

Comment (nemanja-da) [reply testing-daml-models/testing-with-scenarios/5_scenario_tracing.md]: Dunno really, best to look at their [GitHub repo](https://github.com/katacoda/scenario-examples). Just sent an email asking them for help (you're cced)

Comment (ghost) [reply testing-daml-models/testing-with-scenarios/5_scenario_tracing.md]: fixed now.

File: testing-daml-models/testing-with-scenarios/0_intro.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+scenarios are a good way to unit test your DAML model and make sure it stays correct in further
+iterations.
+
+This tutorial will teach you how to
+
+1. write DAML scenarios 
+1. add assertions to you scenarios
+1. trace through your scenarios and output debug information
+
+We'll take the DAML model of the social network of the [GettingStarted](https://katacoda.com/daml/courses/getting-started)
```
Comment (nemanja-da) [inline testing-daml-models/testing-with-scenarios/0_intro.md]:
```suggestion
We'll take the DAML model of the social network of the [GettingStarted](https://daml.com/learn/getting-started/)
```

All scenarios will be available at daml.com/learn. Bernhard updated the Katacoda instructions.

File: testing-daml-models/testing-with-scenarios/5_scenario_tracing.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+      create this with following = userToFollow :: following
+</pre>
+
+with
+
+<pre class="file" data-target="clipboard">
+      trace "adding new follower" (create this with following = userToFollow :: following)
+</pre>
+
+If you look at the transaction view of the scenario result panel now, you see your trace at the very
```
Comment (nemanja-da) [inline testing-daml-models/testing-with-scenarios/5_scenario_tracing.md]:
```suggestion
Make sure that `trace` has the same indentation as `archive self`. If you look at the transaction view of the scenario result panel now, you see your trace at the very
```

**Review by nemanja-da — APPROVED** [review]
Thnx a bunch Robin :)

**Pull request merged**